virttest/qemu_devices/qcontainer.py

```diff
@@ -1690,18 +1690,15 @@ def machine_s390_virtio(machine_params):
                 qdevices.QNoAddrCustomBus(
                     "bus",
                     [["addr"], [64]],
```

Comment:
Is the addr length also 64?

Reply:
you are right, I asked developers before, it should be 32